src/app/comments/comments.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+import { faPencilAlt, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import { faComments, faThumbsUp } from '@fortawesome/free-regular-svg-icons';
+
+import { Comment } from './comment.model';
+import { CommentsService } from '../services/comments/comments.service';
+
+@Component({
+  selector: 'app-comments',
+  templateUrl: './comments.component.html',
+  styleUrls: ['./comments.component.css']
+})
+export class CommentsComponent implements OnInit {
+  faComments = faComments;
+  faPencilAlt = faPencilAlt;
+  faTrashAlt = faTrashAlt;
+  faThumbsUp = faThumbsUp;
+  data: any;
+  email = localStorage.getItem("email");
+  liked = false;
+
+
+  commentForm = new FormGroup({
+    content: new FormControl('')
+  });
+
+  constructor(private commentService: CommentsService) { }
+
+  ngOnInit(): void {
+    this.getComments();
+  }
+
+  onSubmit() {
```

Comment:
Tại sao có những function có void có những cái lại không?
```suggestion
  onSubmit(): void { 
```